### PR TITLE
LPS-190517

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://liferay.atlassian.net/browse/LPS-190517

PASSED Testing on: https://github.com/CaiocSiqueira/liferay-portal/pull/51

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase [hunk 1]
```diff
@@ -1617,6 +1617,70 @@ definition {
...
+
+			Publications.publishPublication(publicationName = "Publication Name");
+
+			Publications.viewPublicationPublished(publicationName = "Publication Name");
+		}
+
+		task ("View Publication history and confirm that the changes were made") {
+			PublicationsNavigator.openPublicationsHistory();
+
+			Click(locator1 = "//div[contains(text(),'Publication Name')]");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase:1655]: Would `Publications#PUBLISHED_PUBLICATION_NAME` work instead?

**Comment by xbrianlee** [comment]
@CaiocSiqueira Just one change. After that, you can forward this.

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:forward

**Pull request closed without merging**